rs/canister_sandbox/src/replica_controller/sandbox_process_eviction.rs

```diff
@@ -101,17 +129,21 @@ mod tests {
                 id: canister_test_id(i),
                 last_used: now - Duration::from_secs(i),
                 rss: 0.into(),
+                scheduler_priority: AccumulatedPriority::new(0),
             });
         }
         assert_eq!(
             evict(
                 candidates.clone(),
                 0.into(),
-                100,
+                51,
                 now - Duration::from_secs(50),
                 0.into()
             ),
-            candidates.into_iter().rev().take(49).collect::<Vec<_>>()
+            {
+                candidates.sort_by_key(|x| x.last_used);
+                candidates.into_iter().take(49).collect::<Vec<_>>()
+            }
```

Comment:
No functional changes here, it was just more obvious what is tested when `sort` is added, instead of using the assumption of how candidates are created in order of `last_used` time.